capten/agent/cmd/agent/main.go

```diff
@@ -87,3 +103,21 @@ func configureDB() error {
 	}
 	return nil
 }
+
+func startClaimSync(crontInterval string) (*cron.Cron, error) {
+	cronJob := cron.New()
+
+	fetch, err := captenagentsync.NewFetch()
+	if err != nil {
+		log.Errorf("Failed to initialize the sync: %v", err)
+		return nil, err
+	}
+
+	_, jobErr := cronJob.AddJob(fmt.Sprintf(StrInterval, crontInterval), cron.NewChain(cron.SkipIfStillRunning(cron.DefaultLogger)).Then(fetch))
+	if jobErr != nil {
+		log.Errorf("Failed to add cronJob for sync clusterClaim: %v", jobErr)
+		return nil, jobErr
+	}
+
+	return cronJob, nil
+}
```

Comment:
reuse the schduler.go in vault to add the jobs, its easier and clean to add jobs to scheduler

Reply:
That's available under internal folder, can't be imported in other modules.

Reply:
That's available under internal folder, can't be imported in other modules.

Reply:
reuse the code, not import the module